less/variables.less

```diff
@@ -1004,6 +1004,7 @@
 @panel-heading-color:         @brand-navy-minus-10;
 @panel-border-radius:         @border-radius-base;
 @panel-focus-transtion:       @transition-duration;
+@decision-focus-transition:   @transition-duration;
```

Comment:
Is this the only decision variable?  It should be grouped with the others if there are any.  If the only one, should anything else be made a variable?